nens_auth_client/templates/nens_auth_client/invitation.html

```diff
@@ -0,0 +1,5 @@
+<p>Dear madam, sir,</p>
```

Comment:
Nog een idee om dit neutraler neer te zetten? LGHBTnogwattus enzo. Ik vind het ook erg ouderwets klinken.

Misschien iets voor Lirry of Mees?
Waarschijnlijk meer een TODO dan "nu doen" :-)

Reply:
"Dear user" ?